spec/features/items/index_spec.rb

```diff
@@ -12,18 +12,20 @@
       @dog_bone = @brian.items.create(name: "Dog Bone", description: "They'll love it!", price: 21, image: "https://img.chewy.com/is/image/catalog/54226_MAIN._AC_SL1500_V1534449573_.jpg", active?:false, inventory: 21)
     end
 
-    it "all items or merchant names are links" do
+    it "all items, pictures, and merchant names are links" do
       visit '/items'
 
       expect(page).to have_link(@tire.name)
       expect(page).to have_link(@tire.merchant.name)
+      expect(page).to have_css("img[src*='#{@tire.image}']")
       expect(page).to have_link(@pull_toy.name)
       expect(page).to have_link(@pull_toy.merchant.name)
-      expect(page).to have_link(@dog_bone.name)
-      expect(page).to have_link(@dog_bone.merchant.name)
+      expect(page).to have_css("img[src*='#{@pull_toy.image}']")
+      find(:xpath, "//a/img[@alt='Tug toy dog pull 9010 2 800x800']/..").click
```

Comment:
good job figuring out how to click on image link

Reply:
❤️